docs/manage/fleet/change-network.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Connect a machine to a different network"
```

Comment:
kill me for nit, but 'different network' sounds kind of weird, like it's happening in the context of something else (like troubleshooting?) and you're supposed to know what that something else is?  any reason not to just say 'a new network'?